Propose definition for Digital Blackface (#196)
tatianamac/selfdefined: 11ty/definitions/digital-blackface.md

---

> title: Digital Blackface

Reviewer: ```suggestion
title: digital Blackface
```

slug: digital-blackface
defined: true
Speech: noun
---

> Digital Blackface is a form of appropriation where white and non-black individuals use memes featuring Black people. This is a 21st century version of Black minstrelsy which was prevalent in the 19th and 20th centuries.

Reviewer: ```suggestion
a form of appropriation where non-Black individuals use memes and gifs featuring Black people and even emojis with dark skin; a 21st century version of Black minstrelsy, which was prevalent in the 19th and 20th centuries
```

## Issues

>  This type of action is another in a long history of dehumanizing black individuals into exaggerated versions of themselves, especially black women. Rarely are Black characters given subtle feelings, emotion or actions.

Reviewer: ```suggestion
Digital Blackface is one of many ways of dehumanising Black individuals into exaggerated versions of themselves, especially Black women. Rarely are Black characters given subtle feelings, emotion or actions. Additionally, power comes into it as a non-Black person is using a Black person as a mask (hence where the minstrelsy comes in).

Digital Blackface is also an issue of representation and culture appropriation: memes and gifs are often used in spaces where no real Black people are present, which presents them as a prop to be used in all-white spaces. Black culture is stolen from Black people and commodified by non-Black people.

## Examples

> It can go further than appropration where entire online personas are created. These are easy to spot as they almost always use terms like "as a black woman..." and resort to incorrect and illegitimate use of African American Vernacular English (AAVE).

Reviewer: ```suggestion
- Online personas: It can go further than appropration. Falsified personas are easy to spot as they almost always use terms like "as a black woman..." [sic] and resort to incorrect and illegitimate use of African American Vernacular English (AAVE).[1](https://www.buzzfeednews.com/article/ryanhatesthis/your-slip-is-showing-4chan-trolls-operation-lollipop) 
- Gifs
- Emojis (using a darker-than-your-own skin-tone, especially when you are white)[1](https://www.npr.org/sections/codeswitch/2018/03/21/425573955/white-skin-black-emojis)[2](https://techcrunch.com/2017/10/01/thoughts-on-white-people-using-dark-skinned-emoji/)
```

Simple actions like using dark toned emoji in text or social media posts

## What To Do (instead)

> The easiest solution is to pick white people who you identify with and use memes featuring those people instead.

Reviewer: ```suggestion
If you are not Black, before using a Black person's identity to express or convey an idea, take a moment to reflect upon what you're doing and what message you're trying to convey. In most situations, you would probably be better off picking a gif or meme of a non-Black person, better yet, of a race you identify with.
```

## Further Reading

[https://www.teenvogue.com/story/digital-blackface-reaction-gifs](https://www.teenvogue.com/story/digital-blackface-reaction-gifs)
"For while reaction GIFs can and do every feeling under the sun, white and nonblack users seem to especially prefer GIFs with black people when it comes to emitting their most exaggerated emotions. Extreme joy, annoyance, anger and occasions for drama and gossip are a magnet for images of black people, especially black femmes."

[https://www.theawl.com/2014/08/memes-and-misogynoir/](https://www.theawl.com/2014/08/memes-and-misogynoir/)

> "On Twitter lie countless handles featuring a Black person’s image, run by users who are most assuredly not Black. These accounts, which often include a “ghetto” name — the formula prefix “La+” is a favored trope — are riddled with poor attempts at Black vernacular, and feature stereotypes from the minstrel stage."

Reviewer: The further reading section should be in the front matter, formatted like this. It'll mean that you have to remove the quotes, which I might recommend moving into an earlier section, like Examples to help round them out.:
```
reading 
- text: title of the piece
  href: url
```
(It can go anywhere above the --- lines above)

---
title: digital Blackface
slug: digital-blackface
defined: true
Speech: noun

> ---

Author: ```suggestion
reading:
- text: We Need to Talk About Digital Blackface in Reaction GIFs
  href: https://www.teenvogue.com/story/digital-blackface-reaction-gifs
- text: Memes and Misogynoir
  href: https://www.theawl.com/2014/08/memes-and-misogynoir/
---
```

a form of appropriation where non-Black individuals use memes and gifs featuring Black people and even emojis with dark skin; a 21st century version of Black minstrelsy, which was prevalent in the 19th and 20th centuries

## Issues

Digital Blackface is one of many ways of dehumanising Black individuals into exaggerated versions of themselves, especially Black women. Rarely are Black characters given subtle feelings, emotion or actions. Additionally, power comes into it as a non-Black person is using a Black person as a mask (hence where the minstrelsy comes in).

Digital Blackface is also an issue of representation and culture appropriation: memes and gifs are often used in spaces where no real Black people are present, which presents them as a prop to be used in all-white spaces. Black culture is stolen from Black people and commodified by non-Black people.

## Examples

> - Online personas: It can go further than appropration. Falsified personas are easy to spot as they almost always use terms like "as a black woman..." [sic] and resort to incorrect and illegitimate use of African American Vernacular English (AAVE).[1](https://www.buzzfeednews.com/article/ryanhatesthis/your-slip-is-showing-4chan-trolls-operation-lollipop) 
> - Gifs
> - Emojis (using a darker-than-your-own skin-tone, especially when you are white)[1](https://www.npr.org/sections/codeswitch/2018/03/21/425573955/white-skin-black-emojis)[2](https://techcrunch.com/2017/10/01/thoughts-on-white-people-using-dark-skinned-emoji/)

Author: ```suggestion
- Online personas: It can go further than appropration. Falsified personas are easy to spot as they almost always use terms like "as a black woman..." [sic] and resort to incorrect and illegitimate use of African American Vernacular English (AAVE).[1](https://www.buzzfeednews.com/article/ryanhatesthis/your-slip-is-showing-4chan-trolls-operation-lollipop) 

"On Twitter lie countless handles featuring a Black person’s image, run by users who are most assuredly not Black. These accounts, which often include a “ghetto” name — the formula prefix “La+” is a favored trope — are riddled with poor attempts at Black vernacular, and feature stereotypes from the minstrel stage." [2](https://www.theawl.com/2014/08/memes-and-misogynoir/)

- Gifs
"For while reaction GIFs can and do every feeling under the sun, white and nonblack users seem to especially prefer GIFs with black people when it comes to emitting their most exaggerated emotions. Extreme joy, annoyance, anger and occasions for drama and gossip are a magnet for images of black people, especially black femmes." [1](https://www.teenvogue.com/story/digital-blackface-reaction-gifs)

- Emojis (using a darker-than-your-own skin-tone, especially when you are white)[1](https://www.npr.org/sections/codeswitch/2018/03/21/425573955/white-skin-black-emojis)[2](https://techcrunch.com/2017/10/01/thoughts-on-white-people-using-dark-skinned-emoji/)

```

Simple actions like using dark toned emoji in text or social media posts

## What To Do (instead)

If you are not Black, before using a Black person's identity to express or convey an idea, take a moment to reflect upon what you're doing and what message you're trying to convey. In most situations, you would probably be better off picking a gif or meme of a non-Black person, better yet, of a race you identify with.

> ## Further Reading
> 
> [https://www.teenvogue.com/story/digital-blackface-reaction-gifs](https://www.teenvogue.com/story/digital-blackface-reaction-gifs)
> "For while reaction GIFs can and do every feeling under the sun, white and nonblack users seem to especially prefer GIFs with black people when it comes to emitting their most exaggerated emotions. Extreme joy, annoyance, anger and occasions for drama and gossip are a magnet for images of black people, especially black femmes."
> 
> [https://www.theawl.com/2014/08/memes-and-misogynoir/](https://www.theawl.com/2014/08/memes-and-misogynoir/)
> "On Twitter lie countless handles featuring a Black person’s image, run by users who are most assuredly not Black. These accounts, which often include a “ghetto” name — the formula prefix “La+” is a favored trope — are riddled with poor attempts at Black vernacular, and feature stereotypes from the minstrel stage."

Author: ```suggestion
```
